File: Atividade-2/javascript/main.js
const fs = require('fs');
const path = require('path');
const ListaLigada = require('./ListaLigada'); // Supondo que você esteja usando a biblioteca 'linked-list'
const ExecutadorArquivo = require('./ExecutadorArquivo'); // Importando a classe ExecutadorArquivo

class Main {
    static main() {
       
        const path_req = require('path');

        const path = path_req.resolve(__dirname, 'arq-novo.txt');

        const listaLigada = new ListaLigada();
        const executador = new ExecutadorArquivo();

        console.log("Resposta do arquivo (arq-novo.txt):");
        executador.executa(path, listaLigada);
    }
}

Main.main();

File: Atividade-2/python/main.py
from ListaLigada import ListaLigada
from ExecutadorArquivo import ExecutadorArquivo

def main():
   
    path = "Atividade-2/python/arq-novo.txt"

    listaLigada = ListaLigada()

    print("Resposta do arquivo (arq-novo.txt):")
    ExecutadorArquivo.executa(path, listaLigada)

if __name__ == "__main__":
    main()




